src/Microsoft.DotNet.InternalAbstractions/DirectoryPath.cs

```diff
@@ -46,7 +55,7 @@ public override string ToString()
 
         public DirectoryPath GetParentPath()
         {
-            return new DirectoryPath(Directory.GetParent(Path.GetFullPath(Value)).FullName);
+            return new DirectoryPath(Path.GetDirectoryName(Path.GetFullPath(Value)));
```

Comment:
Does this still need the `Path.GetFullPath` call here?

Reply:
Fixed